polygeist/tools/cgeist/Test/CMakeLists.txt

```diff
@@ -14,6 +14,7 @@ list(APPEND MLIR_CLANG_TEST_DEPS
   FileCheck count not
   llvm-config 
   llvm-dis
+  llvm-as
```

Comment:
Why do we need this in `cgeist` tests?

Reply:
`math_funcs.cpp` requires `llvm-as` and was failing on my machine as it is not build by default with `sycl-toolchain`